<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-06M61PI" failures="1" tests="3" name="Test" time="9.863" errors="0" timestamp="2024-12-12T11:35:06 IST">
  <testcase classname="Listneers.OrangeHRMListnners" name="testlogo" time="0.104"/>
  <testcase classname="Listneers.OrangeHRMListnners" name="testUrl" time="0.024">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at Listneers.OrangeHRMListnners.testUrl(OrangeHRMListnners.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testUrl -->
  <testcase classname="Listneers.OrangeHRMListnners" name="homepageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- homepageTitle -->
</testsuite> <!-- Test -->
